src/site/content/en/payments/android-payment-apps-delegation/index.md

```diff
@@ -0,0 +1,408 @@
+---
+layout: post
+title: Providing shipping and contact information from Android payment app
+subhead: |
+  How to update your Android payment app to provide shipping address and payer's contact information with Web Payments APIs.
+authors:
+  - sahel
+date: 2020-07-17
+description: |
+  Learn how to modify your Android payment app to provide the user's selected shipping address as well as contact information when the merchant has requested them via the Payment Request API.
+tags:
+  - payments
+---
+
+Entering shipping address and contact information through a web form can be a
+cumbersome experience for customers. It can cause errors and lower conversion
+rate.
+
+That's why the Payment Request API supports a feature to request shipping
+address and contact information. This provides multiple benefits:
+
+* Users can pick the right address with just a few taps.    
+* The address is always returned in [the standardized
+  format](https://w3c.github.io/payment-request/#paymentaddress-interface).    
+* Submitting an incorrect address is less likely.
+
+Some browsers can provide address data form their autofill storage and make it
+available through the browser's native UI. This functionality can also be
```

Comment:
```suggestion
available through the browser's built-in UI. This functionality can also be
```
I want to avoid using the word `native` to mean different things.